experiment_buddy/experiment_buddy.py

```diff
@@ -174,7 +181,7 @@ def deploy(use_remote, sweep_yaml, proc_num=1) -> WandbWrapper:
     try:
         git_repo = git.Repo(os.path.dirname(hyperparams["__file__"]))
     except git.InvalidGitRepositoryError:
-        raise ValueError("the main file be in the repository root")
+        raise ValueError("the main file be in the repository root, no git init in example folder!")
```

Comment:
is this still an issue with examples as a different repo?

Reply:
not anymore